README.md

```diff
@@ -93,36 +93,37 @@ signature of the `Comparator:compare` method we want to implement later:
 
 ```java
 public class ComparisonInterceptor {
-  public int intercept(Object first, Object second) {
-    return first.hashCode() - second.hashCode();
+  public int compare(Object first, Object second) {
+    return ((Comparable) first).compareTo(second);
   }
 }
 ```
 
 Note that the above interceptor does not depend on any Byte Buddy class. This is good news because none of the classes
-that by Byte Buddy generates require Byte Buddy on the class path! We can now implement the `Comparator` interface by 
-passing an instance of the above class to `MethodDelegation`: 
+that by Byte Buddy generates require Byte Buddy on the class path! We can now implement the `Comparator` interface by
+passing an instance of the above class to `MethodDelegation`:
 
 ```java
 Class<? extends Comparator> dynamicType = new ByteBuddy()
-  .subclass(Comparator.class) 
-  .method(named("compare")).intercept(MethodDelegation.to(new ComparisonInterceptor())
+  .subclass(Comparator.class)
+  .method(named("compare")).intercept(MethodDelegation.to(new ComparisonInterceptor()))
```

Comment:
note that the missing closing brace should be fixed even if this pr is not accepted :-)